Stratis.Bitcoin/Consensus/ConsensusFeature.cs

```diff
@@ -29,6 +29,7 @@ public class ConsensusFeature : FullNodeFeature
 		private readonly Signals signals;
 		private readonly ConsensusLoop consensusLoop;
 		private readonly NodeSettings nodeSettings;
+		private readonly StakeChain stakeChain;
```

Comment:
I like the way StratisConsensusValidator inherits from ConsensusValidator. 
Can you see any way to subclass a new StratisConsensusFeature, and StratisConsensusLoop instead of adding StakeChain to the base class?